shoop/campaigns/models/campaigns.py

```diff
@@ -74,14 +69,9 @@ def is_available(self):
                 return False
         return True
 
-    def save(self, **kwargs):
-        if self.discount_percentage and self.discount_amount_value:
-            raise ValidationError(_("You should only define either discount percentage or amount."))
-
-        if not (self.discount_percentage or self.discount_amount_value):
-            raise ValidationError(_("You must define discount percentage or amount."))
-
-        super(Campaign, self).save(**kwargs)
+    @property
+    def type(self):
```

Comment:
nowhere now, it was added because @suutari-ai said campaigns should have such property.